examples/disk_pressure_read_unsafemode.yaml

```diff
@@ -0,0 +1,25 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: disk-pressure-read
+  namespace: chaos-demo
+spec:
+  level: pod
+  selector:
+    app: demo-curl
+  count: 1
+  unsafeMode:
+    disableAll: false
+    config:
+      countTooLarge:
```

Comment:
Is the purpose of this example to showcase `config.countTooLarge`? Then I think we should either drop the selector entirely or use a very broad one, e.g., `os: linux`, `cloud: aws`, etc., as well as rename the file. Wdyt?